independent-publisher-connectors/PowerBI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1062 @@
+			"x-ms-summary" : "Number of records to return",
+            "description": "Use the top parameter to set the first n results"
+          },
+          {
+            "name": "$skip",
+            "in": "query",
+            "required": false,
+            "type": "string",
+			"x-ms-summary" : "Number of records to skip",
+            "description": "Use the top parameter to skip the first x amount of groups that are being returned."
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Expands related entities inline",
+			"x-ms-summary" : "Enter expand query"
+          }
+        ],
+        "summary": "Get Datasets In Group As Admin",
+        "operationId": "DatasetsGetDatasetsInGroupAsAdmin",
+        "description": "Returns a list of datasets from the specified workspace"
+      }
+    },
+    "/v1.0/myorg/admin/datasets/{datasetId}/datasources": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "type": "string",
+                        "description": "name"
+                      },
+                      "connectionString": {
+                        "type": "string",
+                        "description": "connectionString"
+                      },
+                      "datasourceType": {
+                        "type": "string",
+                        "description": "datasourceType"
+                      },
+                      "datasourceId": {
+                        "type": "string",
+                        "description": "datasourceId"
+                      },
+                      "gatewayId": {
+                        "type": "string",
+                        "description": "gatewayId"
+                      },
+                      "connectionDetails": {
+                        "type": "object",
+                        "properties": {
+                          "server": {
+                            "type": "string",
+                            "description": "server"
+                          },
+                          "database": {
+                            "type": "string",
+                            "description": "database"
+                          }
+                        },
+                        "description": "connectionDetails"
+                      }
+                    }
+                  },
+                  "description": "value"
+                }
+              }
+            }
+          }
+        },
+        "summary": "DatasetsGetDataSourcesAsAdmin",

```

Comment:
@paulculmsee Documentation - [Summary Values](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Reply:
@paulculmsee Can you please fix it